docs/user/integration/opentelemetry-demo/README.md

```diff
@@ -0,0 +1,213 @@
+# Integrate OpenTelemetry Demo App
+
+## Overview
+
+| Category| |
+| - | - |
+| Signal types | traces, metrics |
+| Backend type | custom in-cluster, third-party remote |
+| OTLP-native | yes |
+
+Learn how to install the OpenTelemetry [demo application](https://github.com/open-telemetry/opentelemetry-demo) on a Kyma cluster using a provided [Helm chart](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-demo). The demo application will be configured to push trace data using OTLP to the collector that's provided by Kyma, so that they are collected together with the related Istio trace data.
+
+![setup](./../assets/otel-demo.drawio.svg)
+
+## Table of Content
+
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Advanced](#advanced)
+- [Clean Up](#clean-up)
+
+## Prerequisites
+
+- Kyma as the target deployment environment
+- The [Telemetry module](../../README.md) is [enabled](https://kyma-project.io/#/02-get-started/08-install-uninstall-upgrade-kyma-module?id=install-uninstall-and-upgrade-kyma-with-a-module)
+- kubectl version 1.22.x or higher
+- Helm 3.x
+
+## Installation
+### Preparation
```

Comment:
please fix the numbering also here

Reply:
It just looks weird in the raw text, but each list item being a `1.` is [valid markdown](https://www.markdownguide.org/basic-syntax/#lists-1). Icke prefers it this way.